packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,4 +1,65 @@
+const listOfMetricsSend = new Set()
+
+function debounce(fn, timeout) {
+  let timer = null
+
+  return function (...args) {
+    if (timer) {
+      clearTimeout(timer)
+    }
+
+    timer = setTimeout(fn, timeout, ...args)
+  }
+}
+
+function sendWebVitals(dataLayerName = `dataLayer`) {
+  const win = window
+
+  function sendData(data) {
+    if (listOfMetricsSend.has(data.name)) {
+      return
+    }
+    listOfMetricsSend.add(data.name)
+
+    sendToGoogleAnalytics(data, win[dataLayerName])
```

Comment:
`sendToGoogleAnalytics` doesn't seem to make use of second param - so either we can drop it or make that function use it? Seems like relict of c&p of similar thing from tagmanager plugin?

```suggestion
    sendToGoogleAnalytics(data)
```

and probably can drop `const win = window` too (if we don't actually use it)